metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -47,6 +48,18 @@ record DashboardInfo includes CustomProperties, ExternalReference {
   }
   charts: array[ChartUrn] = [ ]
 
+  /**
+   * Datasets consumed by a dashboard
+   */
+  @Relationship = {
+    "/*": {
+      "name": "Consumes",
+      "entityTypes": [ "dataset" ],
+      "isLineage": true
+    }
+  }
+  datasets: array[DatasetUrn] = [ ]
```

Comment:
one minor comment: we're actually trying to move away from using specific urns in general like this, instead could you use `array[Urn]`?

Reply:
Is done. 😊

https://github.com/datahub-project/datahub/pull/5188/files#diff-6ae54ee7e3babcd8ac0b99538e8de6725f9fd82c0a119e517c20b2a23575e303R61

Reply:
Thanks!